src/util-exception-policy.c

```diff
@@ -73,6 +75,83 @@ void ExceptionPolicyApply(Packet *p, enum ExceptionPolicy policy, enum PacketDro
     SCLogDebug("end");
 }
 
+/* Should I make a separate enum for this? With a separate function?
+   This seems to make sense in my head right now, as then things would be more logically containted,
+   and then I could call the master-switch function from within the ExceptionPolicyParse function...
+ */
+
+void setMasterExceptionPolicy()
+{
+    const char *switch_str = NULL;
+    if (EngineModeIsIPS()) {
```

Comment:
this shouldn't be specific to IPS mode. Performance mode (or bypass mode) would be useful for IDS mode too

Reply:
Ok! So, only the drop portion would be IPS-only?

Reply:
yeah